src/libs/actions/Policy.js

```diff
@@ -887,15 +887,30 @@ function createWorkspace() {
             key: `${ONYXKEYS.COLLECTION.REPORT}${announceChatReportID}`,
             value: {pendingAction: null},
         },
+        {
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${announceChatReportID}`,
+            value: {pendingAction: null},
```

Comment:
This should be set under the key 0 because that's how it's created in `createOptimisticCreatedReportAction`. When I test with this change it works, but not without it.
```javascript
value: {
    0: {
        pendingAction: null,
    },
},
```

Reply:
oh yeah nice, good catch